web/src/ehr/patient/EditPatient.vue

```diff
@@ -2,16 +2,18 @@
   <h1 class="mt-10 mb-6">Edit Patient</h1>
 
   <form @submit.stop.prevent="submit">
-    <div class="p-3 bg-red-100 rounded-md" v-if="formErrors.length">
-      <b>Please correct the following error(s):</b>
-      <ul>
-        <li v-for="error in formErrors">* {{ error }}</li>
-      </ul>
-    </div>
+    <div class="bg-white rounded-lg shadow-lg">
+      <div class="sticky top-0 z-10 p-3 bg-red-100 text-red-500 rounded-t-md" v-if="formErrors.length">
+        <label class="text-red-500">Please correct the following error{{ formErrors.length === 0 ? '' : 's' }}:</label>
 
+        <ul class="text-sm">
+          <li v-for="error in formErrors">— {{ error }}</li>
```

Comment:
Is this somthing to fix with tailwinds list-style e.g. `list-disc` https://tailwindcss.com/docs/list-style-type